RecoMET/METAlgorithms/src/METSignificance.cc

```diff
@@ -20,16 +20,18 @@ Description: [one line class summary]
 
 metsig::METSignificance::METSignificance(const edm::ParameterSet& iConfig) {
 
-  edm::ParameterSet cfgParams = iConfig.getParameter<edm::ParameterSet>("parameters");
+  // if(iConfig.exists("parameters")) {
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant